src/routes/language-page.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import Root from "./root";
+import useFetchData from "../components/fetchData";
+export default function Languages() {
+  const { countries, loading } = useFetchData();
+  if (loading) {
+    <div>
+      <p>Loading...</p>
+    </div>;
+  }
+
+  // map through the api making an array of official_languages, no duplicates
+  const duplicates = countries.map((country) => country.official_languages);
+  const languages = countries
+    .map((country) => country.Languages)
+    .filter(...duplicates);
```

Comment:
Can you explain to me how you're using the spread operator here? This sort of syntax still eludes me in my understanding of it.

Reply:
lol it doesn't work.  I haven't gotten back to it yet.  I think that I'm misusing the spread op here